services/agents/postgresql_test.go

```diff
@@ -30,64 +30,95 @@ import (
 )
 
 func TestPostgresExporterConfig(t *testing.T) {
```

Comment:
In this some subtests modify shared objects. I improved them a bit with setup that is called before each test